pelita/scripts/pelita_main.py

```diff
@@ -208,14 +217,14 @@ def long_help(s):
                     metavar='REPLAYFILE', dest='replayfile', const='pelita.dump', nargs='?')
 parser.add_argument('--store-output', help=long_help('Write all player’s stdout/stderr to the given folder (must exist)'),
                     metavar='FOLDER')
-parser.add_argument('--list-layouts', action='store_true',
-                    help='List all available built-in layouts.')
 parser.add_argument('--check-team', action="store_true",
                     help=long_help('Check that the team is valid (on first sight) and print its name.'))
 parser.add_argument('--append-blue', type=str, metavar='INFO', default=None,
                     help=long_help('Append info about the blue team (such as group id).'))
 parser.add_argument('--append-red', type=str, metavar='INFO', default=None,
                     help=long_help('Append info about the red team (such as group id).'))
+parser.add_argument('--save-layout', help=long_help('Save current layout to a file'),
```

Comment:
60:40 feeling to prefer --store-layout or --write-layout since we need to give a file name. (--save feels more implicit to me like an automatic option that already includes the file name, but maybe that’s just me.)

Reply:
fixed in bbb10cca